websites/N/Nettruyen/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+  "author": {
+    "name": "Kinue",
+    "id": "634246091293851649"
+  },
+  "service": "Nettruyen",
+  "description": {
+    "en": "Biggest manga website in Vietnam with latest update",
+    "vi_VN": "Web đọc truyện tranh online lớn nhất Việt Nam được cập nhật liên tục mỗi ngày"
+  },
+  "url": [
+    "www.nettruyen.com",
+    "nettruyen.com"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/jcR1P82.png",
+  "thumbnail": "https://i.imgur.com/eGhdGi5.png",
```

Comment:
logo must be 512x512 and doesnt look like the logo that is on the website...?

Reply:
uhh ![img](https://stop-simp.hololive.wtf/‍‍⁠‍‍‍‌‌‌​‍‍​⁠⁠​⁠‍​)